### PR TITLE
Add load table call and add test

### PR DESCRIPTION
**Pull request by sfc-gh-aixu**
# Description

Please include a summary of the changes and the related issue. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Documentation update
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B

**Test Configuration**:
* Hardware:
* Toolchain:
* SDK:

# Checklist:

Please delete options that are not relevant.

- [ ] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] If adding new functionality, I have discussed my implementation with the community using the linked GitHub issue

### CODE REVIEW

File: polaris-core/build.gradle.kts [hunk 1]
```diff
@@ -34,6 +34,19 @@ dependencies {
     implementation("io.airlift:aircompressor:0.27") { because("Vulnerability detected in 0.25") }
   }
 
+  implementation("org.apache.xtable:xtable-core_2.12:0.2.0-incubating") {
```
Comment (RussellSpitzer) [inline polaris-core/build.gradle.kts:37]: Why exclude then add back in transitives? Is this just to avoid the log4j conflicts?

**Comment by RussellSpitzer** [comment]
@aihuaxu can you rebase?

File: polaris-core/build.gradle.kts (hunk 1 above)
Comment (sfc-gh-lbao) [reply polaris-core/build.gradle.kts:37]: yep, xtable is using Spark 3.4 and corresponding Delta, which is not compatible with Polaris project. Overriding the dependencies to use Spark 3.5 and Delta 3.2 works.

**Comment by aihuaxu** [comment]
Rebased.

**Review by sfc-gh-yzou — APPROVED** [review]

**Review by sfc-gh-gkaraje — APPROVED** [review]

**Pull request merged**